changelog/unreleased/issue-20925.toml

```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Fix failing of executing search on event definituon page"
```

Comment:
typo

Reply:
I added runPollJob usage. We can refactor all of this and replace with a search implementation usage. But  I would prefer to do it in separate PR.